src/processor/k8sattributesprocessor/options.go

```diff
@@ -187,8 +189,8 @@ func withExtractMetadata(fields ...string) option {
 				p.rules.ContainerImageName = true
 			case conventions.AttributeContainerImageTag:
 				p.rules.ContainerImageTag = true
-			default:
-				return fmt.Errorf("\"%s\" is not a supported metadata field", field)
+			case clusterUID:
```

Comment:
Missing default is from the original?

Reply:
Yes. The error handling is now handled elsewhere.